docs/conf.py

```diff
@@ -110,7 +110,7 @@
 templates_path = ["_templates"]
 html_css_files = ["style.css", "custom.css", "gallery.css"]
 
-nbsphinx_timeout = 2400
+nbsphinx_timeout = 360
```

Comment:
Can we change this back?

Reply:
Once the performance bug was fixed in Terra I put back the old values in all repositories. This helps us identify another bug like that one.